recipe/meta.yaml

```diff
@@ -14,14 +14,20 @@ build:
   entry_points:
     - retriever = retriever.__main__:main
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
-
+  script: 
+    - python setup.py install --single-version-externally-managed --record=record.txt
+ 
 requirements:
   build:
     - python
     - setuptools
     - xlrd
     - future
+    - argcomplete
+    - numpydoc
+    - psycopg2
+    - pytest
+    - pymysql
```

Comment:
Are these all only required for the build or are some (possibly all) of these needed during run time?

Also normally we only add things to the build requirements if they are strictly needed for building. Typically this is `python` and maybe `setuptools` though things from `setup_requires` in `setup.py` may be needed as well. Given this, are all of these needed for the build or could some of these be dropped from `requirements/build`?

Reply:
Thanks @jakirkham so much for review, 
The Core projects imports  are (psycopg2, pymysql, xlrd, future, argcomplete) 
To build (setuptools)
To build docs (numpydoc)


ref: https://github.com/weecology/retriever/blob/master/requirements.txt

```future
xlrd>=0.7
argcomplete
PyMySQL>=0.4
psycopg2>=2.0
numpydoc
sphinx_py3doc_enhanced_theme
sphinxcontrib-napoleon```

Reply:
So are the core project imports being imported or used during the build? If so, it is totally fine to keep them here. If not, maybe it is worth dropping those from the build requirements.